Models/ProtoModel.cpp

```diff
@@ -45,9 +46,12 @@ void ProtoModel::RebuildSubModels() {
         subModels.protoModels[field->number()] = new ProtoModel(refl->MutableMessage(protobuf, field), this);
       }
     } else if (field->cpp_type() == CppType::CPPTYPE_STRING && field->is_repeated()) {
-      for (int j = 0; j < refl->FieldSize(*protobuf, field); j++) {
-        subModels.strings[field->number()].append(QString::fromStdString(refl->GetRepeatedString(*protobuf, field, j)));
-      }
+      if (field->name() == "subimages")
```

Comment:
Good news: we have a file_type extension for this, now. Anyway, like I said on Discord, ignore any of this, but please add TODOs.